R/utils.R

```diff
@@ -207,6 +209,18 @@ parse_database_error <- function(msg) {
   )
 }
 
+contextualize_database_error <- function(cnd_body) {
+  if (any(grepl("Data source name not found", cnd_body))) {
+    cnd_body <-
+      c(
+        cnd_body,
+        "See {.help odbc::odbcListDataSources} to learn more."
```

Comment:
Should this be an `i` bullet?

Reply:
Good point! Required a bit of refactoring that feels overkill for now but will make extensions straightforward.